WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html

```diff
@@ -6,32 +6,37 @@
     </head>
 
     <body>
-        <h1>Your Selection</h1>
-        <div class="detail-block-1">
-            <div class="detail-block-1-1">
-                <img src="{% get_static_prefix %}{{ item.image }}" alt="item image">
-                <div class="detail-block-1-1-pane">
-                    <div class="row-1">{{item.name}}</div>
-                    <div class="row-2">
-                        <div id="price-div" data-price="{{ item.price }}">{{item.price}}</div>
-                        <div id="count-square" class="square"><a id="clicks">1</a></div>
-                        <button id="plus-butn" onClick="onClickPlusOne()">+</button>
-                        <button id="minus-butn" onClick="onClickMinusOne()">-</button>
-                    </div>
-                    <div class="row-3">
-                        <p id="before-total-text">TOTAL:&#65129;<span id="total">{{item.price}}</span></p>
-                    </div>
-                    
-                    <div class="row-4">
-                        <a href='' target="_self">PAY NOW</a>
+        <div id="container">
+            <h1>Your Selection</h1>
+            <div class="detail-block-1">
+                <div class="detail-block-1-1">
```

Comment:
can you choose a more meaningful name here "detail-block-1-1"?

Reply:
the noted reflected in commit # [466c2aa](https://github.com/EsamSharaf/WebCoffeeMachine/pull/5/commits/466c2aa9ec6dcae7b9a799df5b0e12fde64078b9)